app/main/forms/suppliers.py

```diff
@@ -48,42 +48,42 @@ class EditContactInformationForm(Form):
 
 class DunsNumberForm(Form):
     duns_number = IntegerField('DUNS Number', validators=[
-        DataRequired(message="DUNS Number must be 9 digits"),
-        NumberRange(min=100000000, max=999999999, message="DUNS Number must be 9 digits")
+        DataRequired(message="You must enter a DUNS number with 9 digits."),
+        NumberRange(min=100000000, max=999999999, message="You must enter a DUNS number with 9 digits.")
     ])
 
 
 class CompaniesHouseNumberForm(Form):
     companies_house_number = StringField('Companies house number', validators=[
         Optional(),
-        Length(min=8, max=8, message="Companies house number must be 8 characters")
+        Length(min=8, max=8, message="You must enter a Companies House number with 8 characters.")
```

Comment:
That's a good point. This change wasn't in the doc with the others, I just added it to make that sentence conform to the new style.

@superroz can you advise on how we should explain this?